docs/source/developers/overview.rst

```diff
@@ -97,9 +97,9 @@ Pull request and review
 When contributing a patch, use this list as a checklist of Apache Arrow workflow:
 
 * Submit the patch as a **GitHub pull request** against the **master branch**.
-* So that your pull request syncs with the JIRA issue, **prefix your pull request
-  name with the JIRA issue id** (ex:
-  `ARROW-767: [C++] Filesystem abstraction <https://github.com/apache/arrow/pull/4225>`_).
+* So that your pull request syncs with the GitHub issue, **prefix your pull request
+  title with the GitHub issue id** (ex:
+  `GH-14866: [C++] Remove internal GroupBy implementation <https://github.com/apache/arrow/pull/14867>`_).
```

Comment:
Should we keep both examples of prefixing with github issue id or JIRA number? Since until JIRAs are migrated, PRs will be opened targetting both github and jira issues

Reply:
Can do 👍